### PR TITLE
Add example to autoviz

### PR DESCRIPTION
**Pull request by aggFTW**

### CODE REVIEW

File: autovizwidget/examples/autoviz.ipynb [hunk 1]
```diff
@@ -0,0 +1,688 @@
...
+    "![widget2](Capture0.PNG)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Convenience methods\n",
+    "\n",
+    "Alternatively, you can let the library inspect your pandas DataFrame and select the X and Y axis automatically for you.\n",
```
Comment (msftristew) [inline autovizwidget/examples/autoviz.ipynb]: Something to consider (this is a nitpick) -- I might rather present `display_dataframe` as the default, standard way to display a dataframe, and then offer the `Encoding` method as a more low-level way to do this if you need more control.  I'm fine with either though.

Comment (msftristew) [inline autovizwidget/examples/autoviz.ipynb]: My reasoning: this is the `autovizwidget` library, so I think I would expect something to do autoviz without me having to do anything else :)

Comment (aggFTW) [inline autovizwidget/examples/autoviz.ipynb]: Done :)

**Comment by aggFTW** [comment]
Merging. I can address other feedback later.

**Pull request merged**